test/shared/box.lua

```diff
@@ -97,23 +97,23 @@ box.once('initialization', function()
         parts  = {2, compat.unsigned, 3, compat.string}
     })
 
-    local space = box.schema.space.create('msgpack')
-    space:create_index('primary', {
+    local space_msgpack = box.schema.space.create('msgpack')
```

Comment:
I don't see anything bad here.

Reply:
local space - double definition.

Reply:
I see. But I use redefinitions in Lua very often (mostly in tests). I just consider this luacheck warning as false-positive and sometimes ask others to don't change names just because `luacheck` w/o `-r` complains. See related comments:

https://lists.tarantool.org/pipermail/tarantool-patches/2020-June/017304.html
https://lists.tarantool.org/pipermail/tarantool-patches/2020-June/017308.html

Reply:
I disagree with you in this aspect, because IMHO it's a bad practice that makes the code harder to read and, as result, can cause problems during the debugging phase (for example). Something like if() without curve braces in C.
But I don't mind removing this commit from the patchset.

Reply:
Drop the commit.